src/std_extensions/collections.rs

```diff
@@ -126,22 +193,107 @@ fn list_and_elem_type(list_type_def: &TypeDef) -> (Type, Type) {
     );
     (list_type, elem_type)
 }
+
+/// A list operation
```

Comment:
The `collections` extension doesn't seem to be mentioned in the main spec; is there a spec for it somewhere else?

Reply:
Not yet, it is somewhat of a placeholder for adding more list operations. See #725